terraform/datadog/aws.tf

```diff
@@ -78,7 +78,7 @@ resource "datadog_monitor" "ec2-cpu-utilisation" {
 resource "datadog_monitor" "ec2-cpu-utilisation-es" {
   name                = "${format("%s EC2 high CPU utilisation - ElasticSearch", var.env)}"
   type                = "metric alert"
-  query               = "${format("avg(last_1h):avg:aws.ec2.cpuutilization{deploy_env:%s,job:elasticsearch_master} by {bosh-job,bosh-index} > 95", var.env)}"
+  query               = "${format("avg(last_1h):avg:aws.ec2.cpuutilization{deploy_env:%s,bosh-job:elasticsearch_master} by {bosh-job,bosh-index} > 95", var.env)}"
```

Comment:
Is there a reason to change this in this story? There are plenty of other `job:` filters that would need to be changed.